index.html

```diff
@@ -62,5 +62,234 @@ <h2 class="heading_2">
             </li>
         </ul>
     </section>
+    <main class="works">
+        <section class="Title">
+            <h1 class="title">
+                My Recent Works
+            </h1>
+        </section>
+
+        <section class="multi_card">
+            <img src="images/post_stories.png" alt="screenshot of a gymfit landing webpage">
+            <div class="right_block">
+                <h1 class="stories">
+                    Multi-Post Stories
+                </h1>
+                <p class="content">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+                <ul class="tags">
+                    <li class="tag css">css</li>
+                    <li class="tag html">html</li>
+                    <li class="tag bootstrap">bootstrap</li>
+                    <li class="tag ruby">Ruby</li>
+                </ul>
+                <div class="button">
+                    <button class="project" type="submit">
+                        See Project
+                    </button>
+                </div>
```

Comment:
- [ ] Please ensure your project buttons conform to the Figma template, right now there seem to be two shades of orange in  your buttons unlike the Figma design
![image](https://user-images.githubusercontent.com/90556221/178107070-e9beefb7-197e-45ad-92ff-1cbbdb8321f6.png)

Reply:
Hey @zhadier, thanks for the informative review I actually learned a lot from it while i was updating my files. however the buttons behave accordingly when i comment the enabled css rule. I have all the button css rules at the bottom of my style.css file. please clarify.

Reply:
I might've made a mistake then :'). You can ask for a review and the new reviewer will verify

Reply:
Thanks @zhadier , I have just asked for a review now.👍